ha/const.py

```diff
@@ -206,6 +206,9 @@
 RESOURCE_TYPE = "resource_type"
 STONITH_AUTH_TYPE = 'PASSWORD'
 logger_utility_iec_cmd="logger -i -p local3.err"
+CLUSTER_STOP_KEY = "cluster_stop_key"
+CLUSTER_STOP_VAL_ENABLE = "1"
+CLUSTER_STOP_VAL_DISABLE = "0"
```

Comment:
why do we need ENABLE / DISABLE ? 
We just need to see if key exists or not.

Reply:
yes, we can just check key is exist, but some value needs to put for the key.
and if tomorrow we need to implement cancel we can use disable, so I have added it as a confirmation check.
and this if will hit only once in life.

Reply:
As discussed keeping enable value only (as a default value but not checking anywhere) and removing disable key as not using anywhere.